Rules/CommonScripts/KillMessages.as

```diff
@@ -324,3 +399,101 @@ void onRender(CRules@ this)
 		feed.Render();
 	}
 }
+
+void onCommand(CRules@ this, u8 cmd, CBitStream @params)
+{
+	if (isClient())
+	{
+		if (cmd == this.getCommandID("killstreak message"))
+		{
+			u16 player_netid, kill_count;
+
+			if (!params.saferead_u16(player_netid)
+			 || !params.saferead_u16(kill_count))
+			{
+				print("failed to parse killstreak message payload");
+				return;
+			}
```

Comment:
You don't need to use `saferead_u16` because you know what data to expect in the CBitStream. Just use `read_u16` instead.

Reply:
This also applies to line 456.